ENIGMAsystem/SHELL/Universal_System/Resources/spriteinit.cpp

```diff
@@ -49,6 +49,8 @@ namespace enigma
     // Fetch the highest ID we will be using
     int spr_highid;
     if (!fread(&spr_highid,4,1,exe)) return;
+    
+    sprites.resize(spr_highid+1);
```

Comment:
I think you need early return on if (sprcount == 0) above

Reply:
Right

Reply:
It's still possible though that you have sprites even if the count is 0 because if you create and delete a sprite the high id is incremented I think.

Reply:
that high_id is just for resources packed on the exe